Makefile

```diff
@@ -3,18 +3,28 @@ ifndef CNTR_PROG
 	CNTR_PROG = podman
 endif
 
-CNTR_USER=gvegayon
-IMAGE_NAME=cfa-epinow2-pipeline-dependencies:latest
+ifndef TAG
+	TAG = local
+endif
+
+IMAGE_NAME=cfa-epinow2-pipeline
+
+deps:
+	$(CNTR_PROG) build -t $(REGISTRY)$(IMAGE_NAME)-dependencies:$(TAG) -f Dockerfile-dependencies
 
 build:
-	$(CNTR_PROG) build -t $(IMAGE_NAME) -f Dockerfile-dependencies . && \
-	$(CNTR_PROG) tag $(IMAGE_NAME) $(CNTR_USER)/$(IMAGE_NAME)
+	$(CNTR_PROG) build -t $(REGISTRY)$(IMAGE_NAME):$(TAG) \
+		--build-arg TAG=$(TAG) -f Dockerfile
+
+tag:
+	$(CNTR_PROG) tag $(IMAGE_NAME):$(TAG) $(REGISTRY)$(IMAGE_NAME):$(TAG)
+
+interactive:
+	$(CNTR_PROG) run -v$(PWD):/cfa-epinow2-pipeline -it --rm $(REGISTRY)$(IMAGE_NAME):$(TAG)
```

Comment:
what is attach?

Reply:
I usually do either:

```sh
# This echoes the PID to the console and assumes that CMD keeps the container up
docker run -it --rm --env-file .env -v $(PWD):/mnt $(CNTR_USER)/$(IMAGE_NAME)
# Copy/paste the PID here
docker attach $PID
```

or to attach directly to the container CLI

```sh
docker run  --rm -it $(IMAGE_NAME) /bin/bash
```

Reply:
ohh, you mean to be put into the container's command line? When I run this as is, it puts me into bash inside the container, I _think_ because the `CMD` is `CMD ["bash"]`. But we could make it explicit

Reply:
added